op-node/rollup/sequencing/sequencer_chaos_test.go

```diff
@@ -355,7 +368,7 @@ func testSequencerChaosWithSeed(t *testing.T, seed int64) {
 			t.Fatalf("No action scheduled, but also no events to change inputs left")
 		}
 		if ok && testClock.Now().After(nextTime) {
-			testEm.Emit(SequencerActionEvent{})
+			testEm.Emit(SequencerActionEvent{Ctx: event.WrapCtx(context.Background())})
```

Comment:
we could add a helper method `WithBackgroundCtx()` to `event.Ctx` and then do
```suggestion
			testEm.Emit(SequencerActionEvent{}.WithBackgroundCtx())
```
in such situations as a shortcut